src/Components/CartCard/CartCard.js

```diff
@@ -25,24 +29,44 @@ export default function CartCard() {
                 <div className="description1">
                   <p className="item-price">₹ {product.price}</p>
                   <p>
```

Comment:
Suggestion: you can destructure the product at the top so that  you don't have to write it every time

Reply:
If I remove `product` while mapping above and add destructured properties there then how will I access each `product` item which I want to pass to dispatch?

Reply:
Ohh I didn't see that , you can leave it as it is then